proposals/low-level-struct-improvements.md

```diff
@@ -514,28 +514,24 @@ Detailed Notes:
     - Any member that is not declared on a `struct`
     - Any `static` member, `init` member or constructor on a `struct`
 
-### LifetimeAnnotationAttribute
-The `scoped` and `unscoped` annotations will be emitted into metadata via the type `System.Runtime.CompilerServices.LifetimeAttribute` attribute. This attribute will be matched by name meaning it does not need to appear in any specific assembly.
+### ScopedRefAttribute
+The `scoped` and `unscoped` annotations will be emitted into metadata via the type `System.Runtime.CompilerServices.ScopedRefAttribute` attribute. This attribute will be matched by name meaning it does not need to appear in any specific assembly.
 
 The type will have the following definition:
 
 ```c#
-[AttributeUsage(AttributeTargets.Struct | AttributeTargets.Method | AttributeTargets.Property | AttributeTargets.Parameter)]
-public sealed class LifetimeAnnotationAttribute : Attribute
+namespace System.Runtime.CompilerServices
 {
-    private int _value;
-    public bool IsScoped => _value == 1;
-
-    public LifetimeAnnotationAttribute(int value)
+    [AttributeUsage(AttributeTargets.Parameter, AllowMultiple = false, Inherited = false)]
+    public sealed class ScopedRefAttribute : Attribute
     {
-        _value = value;
     }
 }
 ```
 
 The attribute uses `int` instead of `bool` to allow for future lifetime expansions in this space.
 
-The compiler will emit this attribute on the element targeted by the `scoped` or `unscoped` syntax. This is true for types, methods and parameters. This will only be emitted when the syntax causes the value to differ from its default state. For example `scoped out` will cause no attribute to be emitted.
+The compiler will emit this attribute on the element targeted by the `scoped` syntax. This is true for types, methods and parameters. This will only be emitted when the syntax causes the value to differ from its default state. For example `scoped out` will cause no attribute to be emitted.
```

Comment:
>The compiler will emit this attribute on the element targeted by the `scoped` syntax. This is true for types, methods and parameters. [](http://example.com/codeflow?start=0&length=133)

The second sentence is incorrect since this only applies to parameters. Perhaps change the first two sentences to: "The compiler will emit this attribute on the `scoped` parameter."